MLS.Agent.Tools/Dotnet.cs

```diff
@@ -32,13 +33,13 @@ public Task<CommandLineResult> New(string templateName, string args = null, Budg
             return Execute($@"new ""{templateName}"" {args}", budget);
         }
 
-        public Task<CommandLineResult> AddPackage(string packageId, string version = null, Budget budget = null)
+        public async Task<AddPackageResult> AddPackage(string packageId, string version = null, Budget budget = null)
```

Comment:
I think this can also just be an extension method since it doesn't rely on any non-public state or APIs of the `DotNet` class.